src/components/Detail.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react'
+import '../Detail.css'
+import boyimage from './assets/images/boyimage.png';
+import girlimage from './assets/images/girlimage.png';
+
+class Detail extends React.Component{
+
+    render(){
+        return(
+         <div className="detail container">
+
+            <div className="container maintainers" >
+            <h2>
+                    Project Maintainers
+                </h2>
+                <div className="row">
+
+                </div>
+                <div className="row">
+                <div className="col-sm-12 col-md-6" align="center">
+                    <div className="card text-center">
+                    <img className="card-img-top"  src={boyimage}  alt="Card image cap"></img>
+                    <div className="card-body">
+                        <p className="card-text" ><b>Anush Bhatia</b></p>
+                        <p className="card-text label"><i>Designer</i></p>
+                    </div>
+                    </div>
```

Comment:
We can make this into a seperate card component taking the details as props , it's repeated a lot of times.